### ISSUE TITLE
 Module `AccessibilityInfo` does not exist in the Haste module map

### ISSUE THREAD

**Issue by Tim1437** [reactnativecn/react-native-pushy#224]
运行 pushy bundle --platform android

报以下错误
Bundling with React Native version:  0.57.1
Loading dependency graph, done.
Loading dependency graph...Error: Unable to resolve module `AccessibilityInfo` from `D:\_Work\Web_Study\ReactNative_Study\TestRN\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

使用的版本：
"react-native-update": "^5.1.3"

使用这个命令正常
pushy uploadApk android/app/build/outputs/apk/app-release.apk

**Comment by sofwu** [comment]
是的ios也一样+1

**Comment by ckgan88** [comment]
+ 1

**Comment by sunfan1** [comment]
没有答案么！！？？

**Comment by sunnylqm** [comment]
暂不支持0.57版本，请等候修复

**Comment by sofwu** [comment]
@sunnylqm 好了吗

**Comment by sofwu** [comment]
@sunnylqm 0.57版本修复好了吗？

**Comment by spa221** [comment]
貌似还没好

**Comment by spa221** [comment]
这个如果没修复，是不是就不能进行热更新了？还是直接适用pushy就可以

**Comment by spa221** [comment]
这个没修复是不是就不能进行热更新了，还是0.57版本可以绕过去可以直接适用pushy publish就可以了

**Comment by sofwu** [comment]
@spa221不能用热更新

**Comment by spa221** [comment]
> @spa221不能用热更新

谢谢

**Comment by spryti** [comment]
@sunnylqm 0.57版本修复好了吗？

**Comment by myvary** [comment]
> 暂不支持0.57版本，请等候修复

0.57修复好了吗？

**Comment by runnin86** [comment]
0.57版本什么时候能支持好？

**Comment by gongwenyi** [comment]
同样的问题

**Comment by sunnylqm** [comment]
应该在5.1.4中修复了
